docs/src/pages/components/ComboBox.svx

```diff
@@ -3,7 +3,7 @@
   import { Button } from "carbon-components-svelte";
   import Preview from "../../components/Preview.svelte";
   let comboboxComponent
-
+  let selectedIndex = 1
```

Comment:
Is this line needed?

Reply:
I think it is necessary because of `bind:selecetedIndex`, it's a two-way binded value.